src/dev/src/app/timeline/timeline.demo.ts

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) 2016-2019 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+import { Component } from '@angular/core';
+
+import { ClarityIcons } from '@clr/icons';
+
+@Component({
+  selector: 'clr-timeline-demo',
+  styleUrls: ['./timeline.demo.scss'],
```

Comment:
The `selector` and `styleUrls` can be removed, as well as the timeline.demo.scss file since its empty.

Reply:
All removed.